chore: note that client provided ids are query params for REST (#944)
googleapis/aip: aip/general/0133.md

    duplicate resource, the service **must** error with `PERMISSION_DENIED`
    instead.

**Note:** When supplying a user-specified ID, while the ID is set on
the request message for RPCs, for REST APIs, the `{resource}_id` field

> could be included as a query parameter when making the the request.

Reviewer: > "could be"

I think "will be" is more accurate per the https://google.aip.dev/127 definition of transcoding where anything in the request not in the `body` or path will be in the query parameters. For Create, since the resource is the `body`, the `resource_id` will end up in the query params.

Author: Ack. I think your suggested edit, which I accepted, makes this comment obsolete.

> **Note:** When supplying a user-specified ID, while the ID is set on
> the request message for RPCs, for REST APIs, the `{resource}_id` field
> could be included as a query parameter when making the the request.
> 

Reviewer: ```suggestion
**Note:** For REST APIs, the user-specified ID field, `{resource}_id`, 
will appear in the query parameters of the request.

```

I think we can tighten this up. No need to qualify "RPC" style APIs, we can just state "for this scenario, this will happen"
